src/services/random.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * @fileoverview
+ * Random Service
+ * 
+ * Implements the Arcs Random number service.
+ */
+
+import {Services} from '../runtime/services.js';
+import {Random} from '../runtime/random.js';
+
+Services.register('random', {
+  next: () => Random.next()
```

Comment:
Maybe add a `nextInt()` function as well, based on `Random.next()`? 

As someone who doesn't know the JS conventions for random, this would make it clear to me that `next` returns something on the interval [0, 1) due to the context of `nextInt`.

Reply:
Just a thought. Really, an optional suggestion.

Reply:
I wonder if we can make it easier to document services?  I was going to put more details in the `@fileoverview` section, but it seems like no one would really find it there...

Also think there should be some precondition tests to make sure that no values are passed in since we don't have argument checking...

Reply:
Fwiw, your `@fileoverview` section is right where I would look. 

When/if we have some rendered docs, presumably there would be a way to call up the list of services?

Reply:
Yeah, I think we should be thinking about how we want to organize our Docs, I do prefer auto-generation from source wherever possible..